udata/tests/test_routing.py

```diff
@@ -83,6 +83,11 @@ def model_tester(model):
             assert isinstance(model, self.model)
             return 'ok'
 
+        @app.route('/wrong_model/<tester:model>')
+        def wrong_model_tester(model):
+            assert isinstance(model, self.model)
+            return str(model.id)
```

Comment:
maybe use this `return` in the previous function, and remove this one. I don't see why we would break something else, `ok` is probably not used.

Reply:
Done